### ISSUE TITLE
podman logs: fails on created (but not started) container

### ISSUE THREAD

**Issue by edsantiago** [projectatomic/libpod#433]
Minor corner case:
```
# podman create --name foo fedora date
7486b62d2316429c94bd547b19f2b809e9905d83c709cc0ba478768548003fe2

# podman logs foo
unable to read container log file: open /var/lib/containers/storage/overlay-containers/7486b62d2316429c94bd547b19f2b809e9905d83c709cc0ba478768548003fe2/userdata/ctr.log: no such file or directory
# echo $?
125
```
(works as expected after `podman start foo`). 

FWIW docker simply emits no output and exits 0, which seems like reasonable behavior to me.

**Comment by mheon** [comment]
We should detect when the container is `ContainerStateConfigured` and exit cleanly instead of trying to read the log file to prevent this.

**Comment by edsantiago** [comment]
@mheon sometimes it seems like you reply before I've even hit the Submit button

**Comment by mheon** [comment]
@edsantiago I'm pretty on the spot with github emails. Otherwise they start multiplying in my inbox.

**Comment by mheon** [comment]
Closed via #434
